src/js/states/mods.js

```diff
@@ -77,11 +82,13 @@ export class ModsState extends TextualGameState {
                     <div class="mainInfo">
                         <span class="name">${mod.metadata.name}</span>
                         <span class="description">${mod.metadata.description}</span>
-                        <a class="website" href="${mod.metadata.website}" target="_blank">${T.mods.modWebsite}</a>
+                        <a class="website" href="${mod.metadata.website}" target="_blank">${
+                T.mods.modWebsite
+            }</a>
                     </div>
                     <span class="version"><strong>${T.mods.version}</strong>${mod.metadata.version}</span>
                     <span class="author"><strong>${T.mods.author}</strong>${mod.metadata.author}</span>
-                    <div class="value checkbox checked">
+                    <div class="value checkbox ${!mod.disabled ? "checked" : ""}" id="${mod.metadata.id}:tog">
```

Comment:
colon might not be the most advisable, since it makes selectors have to escape it